Plugins/GitHub3/GitHub3Plugin.cs

```diff
@@ -67,19 +69,26 @@ public static string OAuthToken
 
     [Export(typeof(IGitPlugin))]
     [Export(typeof(IRepositoryHostPlugin))]
-    public class GitHub3Plugin : GitPluginBase, IRepositoryHostPlugin
+    [Export(typeof(IGitPluginForCommit))]
+    public class GitHub3Plugin : GitPluginBase, IRepositoryHostPlugin, IGitPluginForCommit
     {
+        private const int MaxMostRecenetIssuesDisplayed = 10;
+
         private readonly TranslationString _viewInWebSite = new("View in {0}");
         private readonly TranslationString _tokenAlreadyExist = new("You already have an personal access token. To get a new one, delete your old one in Plugins > Plugin Settings first.");
         private readonly TranslationString _generateToken = new("Generate a GitHub personal access token");
         private readonly TranslationString _manageToken = new("Manage GitHub personal access token");
         private readonly TranslationString _openLinkFailed = new("Fail to open the link. Reason: ");
         private readonly TranslationString _noteRestartNeeded = new("Note: Git Extensions need to be restarted so that the token is taken into account.");
+        private readonly TranslationString _noTokenError = new("No GitHub personal access token (PAT) defined");
+        private readonly TranslationString _noAssignedIssues = new("No assigned GitHub issues found");
+        private readonly TranslationString _error = new("Error");
 
         public static string GitHubAuthorizationRelativeUrl = "authorizations";
         public static string UpstreamConventionName = "upstream";
         public readonly StringSetting GitHubHost = new("GitHub (Enterprise) hostname", "github.com");
         public readonly StringSetting PersonalAccessToken = new("OAuth Token", "Personal Access Token", "");
+        private readonly BoolSetting _issueCommitMessageHelperEnabled = new("IssueCommitMessageHelperEnabled", "Enable commit message issue helper", true);
```

Comment:
On by default, probably better (even if got other info than I expected)

Reply:
I was not planning to enable it by default (and I don't explain/remember why I put true in the PR...).
Is everyone ok if I change it to false or still prefer to enable it par default?

Reply:
It is sometimes good to enable functionality by default so the features are found.
This could be very useful for some scenarios, so false will hide the feature (also for existing users if we change later).

But I am OK with false

Reply:
Ok, let's keep it like that...